charts/armonik/values.yaml

```diff
@@ -0,0 +1,538 @@
+############################################
+################## GLOBAL ##################
+############################################
+global:
+  image:
+    pullPolicy: IfNotPresent # FOR KEDA IMAGES
+    registry: "" # FOR KEDA IMAGES
+  imageRegistry: "" # FOR BITNAMI IMAGES AND ARMONIK IMAGES
```

Comment:
What is the difference between `global.image.registry` and `global.imageRegistry`?